### PR TITLE
connection map content added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/dashboards-new/panels/map-charts.md [hunk 1]
```diff
@@ -70,3 +61,38 @@ To add a panel with a map chart:
...
+
+
+:::note
+* Map charts have a display limit of 10,000 results.
+* Colors of map markers cannot be changed.
+:::
+
+## Connection map
+
+Connection map depicts the points of origin of potential threats and the target destinations. This map helps you to visualize the relationships between these sources and destinations, providing insights into the pathways that threats may follow. By visualizing, the connections between threat sources and target destinations, you can enhance your risk assessment, threat modelling, and decision-making processes.
```
Comment (jpipkin1) [inline docs/dashboards-new/panels/map-charts.md:73]:
```suggestion
Connection map depicts the points of origin of potential threats and the target destinations. This map helps you to visualize the relationships between these sources and destinations, providing insights into the pathways that threats may follow. By visualizing the connections between threat sources and target destinations, you can enhance your risk assessment, threat modelling, and decision-making processes.
```

File: docs/dashboards-new/panels/map-charts.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
 ---
 id: map-charts
 title: Map Charts
-description: Map charts are useful for showing IP addresses from log messages on a map.
+description: Map charts helps you to understand the data patterns, trends, and distribution of the data globally.
```
Comment (kimsauce) [inline docs/dashboards-new/panels/map-charts.md:4]:
```suggestion
description: Map charts help you to understand the data patterns, trends, and distribution of the data globally.
```

File: docs/dashboards-new/panels/map-charts.md [hunk 3]
```diff
@@ -70,3 +61,38 @@ To add a panel with a map chart:
 1. Click the **Add to Dashboard** button on the top right of the window to add the panel to your dashboard.  
 
     ![Add to Dashboard button.png](/img/dashboards-new/create-dashboard-new/Add-to-Dashboard-button.png)
+
+
+:::note
```
Comment (kimsauce) [inline docs/dashboards-new/panels/map-charts.md:66]:
```suggestion
:::note limitations
```

**Review by kimsauce — DISMISSED** [review]
Great job, just minor stuff

**Pull request merged**